frontend/src/libs/components/chat-sidebar/chat-element/chat-element.tsx

```diff
@@ -0,0 +1,47 @@
+import { PlainSvgIcon } from '#libs/components/components.js';
+import { IconNameToIcon } from '#libs/enums/enums.js';
+import { getValidClassNames } from '#libs/helpers/helpers.js';
+import { useCallback } from '#libs/hooks/hooks.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+  chat: Chat;
+};
+
+type Chat = {
+  id: number;
+  name: string;
+};
+
+// mocked data
+const selectedChat = {
+  id: 1,
+};
```

Comment:
Name mocked data, starting with mocked, and then we don't need this comment
```suggestion
const mockedSelectedChat = {
  id: 1,
};
```

Reply:
done